sdk/core/core-client-rest/review/core-client.api.md

```diff
@@ -88,8 +88,10 @@ export interface FullOperationResponse extends PipelineResponse {
 // @public
 export function getClient(endpoint: string, options?: ClientOptions): Client;
 
+// Warning: (ae-forgotten-export) The symbol "TracerCallback" needs to be exported by the entry point index.d.ts
+//
 // @public
-export function getClient(endpoint: string, credentials?: TokenCredential | KeyCredential, options?: ClientOptions): Client;
+export function getClient(endpoint: string, credentials?: TokenCredential | KeyCredential, options?: ClientOptions, tracer?: TracerCallback): Client;
```

Comment:
Is this intentional? We cannot add random properties to our core public APIs without design, cross-language consistency, and archboard reviews

Reply:
I think this is absolutely needed.   I got to find a way to pass the tracing logic from the inference to the post function in core-client-rest.    I am having a SDK review meeting on Tuesday.   I will bring up this topic.